ImplementationGuide/markdown/Standort/Standort_Motivation.md

```diff
@@ -0,0 +1,15 @@
+### Motivation
+
+Die Abbildung von Standorten in einem Krankenhaus unterstützt die Raum- und Bettenbelegung in strukturierter Form. 
```

Comment:
"u.a." ... natürlich ist mehr möglich.
Abgrenzung zu Bettenbelegung in DEMIS?

Reply:
Was wären weitere Use-Cases?

Reply:
Ich würde davon ausgehen, dass die Raum- und Bettenbelegung Bestandteil der Funktionalität eines KIS. Was wären Use-Cases um diese Information über eine Schnittstelle zu kommunizieren?